com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -170,6 +173,18 @@ private void ComputeAttachTransform(IXRSelectInteractable interactable)
             {
                 rotationToApply = PlayspaceUtilities.ReferenceTransform.rotation * device.deviceRotation.ReadValue();
             }
+            else
+            {
+                // If we don't have a valid device pose, let's use the palm pose; closest thing we've got!
+                if (handsAggregator == null)
+                {
+                    handsAggregator = HandsUtils.GetSubsystem();
+                }
+                if (handsAggregator != null && handsAggregator.TryGetJoint(TrackedHandJoint.Palm, handController.HandNode, out HandJointPose palmPose))
+                {
+                    rotationToApply = palmPose.Rotation;
```

Comment:
Do we still need to apply `PlayspaceUtilities.ReferenceTransform.rotation *` like we do up on line 174?

Reply:
I suppose it'll be a less common case, as the playspace is moving much less often on devices with hands than with controllers, but I could see a case on something like Quest where you're swapping back and forth between the two

Reply:
Ah damn you're absolutely right. Excellent catch.

Reply:
Done